tool/src/main/java/org/datacommons/tool/Processor.java

```diff
@@ -52,12 +55,17 @@ public void processTables(
     logger.debug("TMCF " + tmcfFile.getName());
     for (File csvFile : csvFiles) {
       logger.debug("Checking CSV " + csvFile.getPath());
-      logCtx.setLocationFile(csvFile.getName());
       TmcfCsvParser parser =
           TmcfCsvParser.init(tmcfFile.getPath(), csvFile.getPath(), delimiter, logCtx);
       Mcf.McfGraph g;
       long numNodesProcessed = 0, numRowsProcessed = 0;
       while ((g = parser.parseNextRow()) != null) {
+        McfMutator mutator = new McfMutator(g.toBuilder(), logCtx);
+        g = mutator.apply();
+
+        McfChecker checker = new McfChecker(g, logCtx);
```

Comment:
echo to the other comment, here feels more nature to do 

McfChecker.check(g, logCtx)

Reply:
Done